src/commands/up/index.ts

```diff
@@ -98,7 +98,7 @@ function runFrontend (dependencies: string) {
   try {
     logger.info('Launching frontend on localhost:3000');
     const commands = [
-      `docker build --pull --build-arg RUNTIME_DEPENDENCIES=${dependencies} -f ${UI_FILEPATH} -t ops-frontend . || exit 1`,
+      `docker build --pull --build-arg RUNTIME_DEPENDENCIES=${dependencies} -f ${UI_FILEPATH} -t ops-frontend .`,
       `docker container stop ops-frontend ${EXIT_0}`,
```

Comment:
Are we missing an `||` here?

Reply:
I'm not too sure, this is to address an issue on windows where it seems like warnings thrown on docker build are causing an exit 1 and the rest of the commands to fail. I think @zsimjee and I are expecting it not to fail with 1 if that `||` is removed